interview-questions.md

```diff
@@ -4,65 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is a block of code with a clear boundary. This block of code is used to perform a specific task or bit of functionality. 
```

Comment:
Yep! I would probably also add that an additional benefit of using a function is that its reusable!